src/Files.App/Interacts/BaseLayoutCommandImplementationModel.cs

```diff
@@ -84,9 +84,16 @@ public virtual void RenameItem(RoutedEventArgs e)
 
 		public virtual async void CreateShortcut(RoutedEventArgs e)
 		{
+			var currentPath = associatedInstance.FilesystemViewModel.WorkingDirectory;
+			if (App.LibraryManager.TryGetLibrary(currentPath, out var library) &&
+				!library.IsEmpty)
+			{
+				currentPath = library.DefaultSaveFolder;
+			}
+
 			foreach (ListedItem selectedItem in SlimContentPage.SelectedItems)
 			{
-				var filePath = Path.Combine(associatedInstance.FilesystemViewModel.WorkingDirectory,
+				var filePath = Path.Combine(currentPath,
 								string.Format("ShortcutCreateNewSuffix".GetLocalizedResource(), selectedItem.Name) + ".lnk");
```

Comment:
The file name could be composed in another variable, and then combine in `filePath` to gain a bit of readability.

Reply:
Done